custom_components/powercalc/data/wiz/9290022650/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script. Controller: hass, Bulbs: 1, Dummy Load: None",
+    "measure_device": "Kasa, KP115(US)",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v0.24.8:docker"
+    },
+    "name": "Philips A19 Warm White Dimmable WiFi LED (Type: SHDW1, Model: 9290022650)",
+    "standby_power": 0.42,
+    "supported_modes": [
+        "lut"
+    ]
```

Comment:
Using `aliases` would be a better solution here. You can add `SHDW1` as alias. Than remove the separate directory.

Reply:
Done.